App.js

```diff
@@ -0,0 +1,159 @@
+// List of speaker objects
+
+const speaker1 = {
+  imageSrc: 'images/Bitpanda.jpg',
+  speakerName: 'Eric Demuth',
+  title: 'CEO and Co-Founder of BitPanda',
+  speakerDescription:
+    "Eric Demuth is Co-founder and CEO of Bitpanda, one of Europe's fastest growing FinTechs, Austria's first unicorn company with a €3.54B valuation.",
+};
+const speaker2 = {
+  imageSrc: 'images/bryan.jpg',
+  speakerName: 'Bryan Che',
+  title: 'CSO Huawei',
+  speakerDescription:
+    'Bryan Che is Chief Strategy Officer at Huawei. There, he leads Huawei’s vision and strategy across its overall businesses and portfolio',
+};
+const speaker3 = {
+  imageSrc: 'images/Cassie.jpg',
+  speakerName: 'Cassie Kozyrkov',
+  title: 'Google Chief Decision Scientist',
+  speakerDescription:
+    'Cassie Kozyrkov is a South African data scientist and statistician. She founded the field of Decision Intelligence at Google, where she serves as Chief Decision Scientist.',
+};
+const speaker4 = {
+  imageSrc: 'images/Michele.jpg',
+  speakerName: 'Michele Romanow',
+  title: 'Co-Founder at Clearco',
+  speakerDescription:
+    'Michele Romanow is a Canadian tech entrepreneur. She co-founded Clearbanc, a Toronto based provider of revenue sharing solutions to fund new online businesses.',
+};
+const speaker5 = {
+  imageSrc: 'images/david.jpg',
+  speakerName: 'David Singleton',
+  title: 'CTO at Stripe',
+  speakerDescription:
+    'David Singleton is chief technology officer at Stripe, where he is responsible for guiding its engineering and design teams as they build economic infrastructure for the internet.',
+};
+const speaker6 = {
+  imageSrc: 'images/Eliza.jpg',
+  speakerName: 'Elizabeth Bramson-Boudreau',
+  title: 'CEO and Publisher of MIT Technology',
+  speakerDescription:
+    'Elizabeth Bramson-Boudreau is the CEO and publisher of MIT Technology Review, the Massachusetts Institute of Technology’s independent media company.',
+};
+
+const speakers = [speaker1, speaker2, speaker3, speaker4, speaker5, speaker6];
+// Target open and close triggers in html
+
+const openEls = document.querySelectorAll('[data-open]');
+const closeEls = document.querySelectorAll('[data-close]');
+const openElems = Array.from(openEls);
+const closeElems = Array.from(closeEls);
```

Comment:
- [ ] [OPTIONAL] `Array.from` is unnecessary in this case since the `querySelectorAll` already returns an object that is iterable. you wouldn't need to convert it to an array before iterating through it.

this means you wouldn't need the extra variables `openElems` & `closeElems` as well.

```suggestion
```